### PR TITLE
Rename Volume DF table header terms

### LINKED ISSUE #655: Quick win: Volume DF -> DCV, tweak column names, tooltip text, column order
### Background / motivation

Here's the status quo of https://df.oceandao.org/volume-df, "DCV" selection:
Screenshot 2023-11-01 at 10 10 59

The tooltips are very generally very helpful. The column names are alright, but we can make some tweaks to greatly help understanding. Quick win FTW:)

Getting a bit more specific, some column names are confusing, and others inconsistent. Tooltips could use some tweaks. And we can tweak the column order.

### Precise Issues, and TODOs

Here are the precise issues, with precise fixes.

**"Rnd5 Avg DCV" label + tooltip "Average Data Consume Volume of a Dataset for the last 5 Data Farming rounds"**
- Problem: "last" implies "this is the final rounds _ever_." (Common English grammar mistake).
- [x] To fix: tooltip "last" -> "previous"
- Problem: "Rnd5 Avg DCV"  implies "Data Farming Round 5". 
- [x] To fix: change label to "Prev5 Avg DCV"

**"Rnd5 Avg DCV" label ...**
- Problems: like "Rnd5 Avg DCV" 
- [x] To fix: tooltip "last" -> "previous"
- [x] To fix: change label to "Prev3 Avg DCV"

**"Rnd Volume" label + tooltip "Total Data Consume Volume of a Dataset for the current Data Farming round"**
- Problem: "Volume" is used, inconsistent with "DCV" everywhere else
- Problem: "Rnd" doesn't imply current round
- [x] To fix: change label to "Cur DCV"

**"Last Rnd Volume" label + tooltip "Total Data Consume Volume of a dataset for the previous round of Data Farming"**
- Problem: in label, "last" grammar mistake
- [x] To fix: change label to "Prev DCV"
- Problem: tooltip text is a bit inconsistent with other ones involving previous rounds. We have an opportunity for better consistency, to ease understanding
- [x] To fix: change tooltip text to "Data Consume Volume of a Dataset for the previous Data Farming round"

**Column ordering**
- Problem: order left to right is: prev 5, prev 3, current, prev 1. It's a confusing order because "current" is in wrong place
- [x] To fix: swap the order of current <> prev 1. Such that the order left to right is: prev 5, prev 3, prev 1, current.

### PR DESCRIPTION
**Pull request by KatunaNorbert**
Fixes #655.

### CODE REVIEW

File: src/utils/metadata/descriptions.json
Comment (idiom-bytes) [inline src/utils/metadata/descriptions.json:12]: Change from "last" => "previous"

File: src/utils/metadata/descriptions.json
Comment (idiom-bytes) [inline src/utils/metadata/descriptions.json:20]: Change from "last" => "previous"

**Review by idiom-bytes — CHANGES_REQUESTED** [review]
Couple of details left to complete all feedback

**Comment by KatunaNorbert** [comment]
@idiom-bytes has requested some changes and needs to give approval before I can merge

**Review by idiom-bytes — APPROVED** [review]
lgtm

**Pull request merged**
